tests/wandb_settings_test.py

```diff
@@ -136,3 +137,88 @@ def test_bad_choice():
         s.mode = "goodprojo"
     with pytest.raises(TypeError):
         s.update(mode="badpro")
+
+
+def test_prio_update_ok():
+    s = Settings()
+    s.update(project="pizza", _source=s.Source.ENTITY)
+    assert s.project == "pizza"
+    s.update(project="pizza2", _source=s.Source.PROJECT)
+    assert s.project == "pizza2"
+
+
+def test_prio_update_ignore():
+    s = Settings()
+    s.update(project="pizza", _source=s.Source.PROJECT)
+    assert s.project == "pizza"
+    s.update(project="pizza2", _source=s.Source.ENTITY)
+    assert s.project == "pizza"
+
+
+def test_prio_update_over_ok():
+    s = Settings()
+    s.update(project="pizza", _source=s.Source.PROJECT)
+    assert s.project == "pizza"
+    s.update(project="pizza2", _source=s.Source.ENTITY, _override=True)
+    assert s.project == "pizza2"
+
+
+def test_prio_update_over_both_ok():
+    s = Settings()
+    s.update(project="pizza", _source=s.Source.PROJECT, _override=True)
+    assert s.project == "pizza"
+    s.update(project="pizza2", _source=s.Source.ENTITY, _override=True)
+    assert s.project == "pizza2"
+
+
+def test_prio_update_over_ignore():
+    s = Settings()
+    s.update(project="pizza", _source=s.Source.ENTITY, _override=True)
+    assert s.project == "pizza"
+    s.update(project="pizza2", _source=s.Source.PROJECT, _override=True)
```

Comment:
I'm a bit confused by `_source` and `_override`.  I assume source indicates where the setting is coming from so PROJECT means we got project specific settings from the API and ENTITY overrides PROJECT level settings.  

I would assume `_override` means ignore the source hierarchy but that doesn't seem to be the case.

Reply:
See more comments below, I found your comment on `override`.

Reply:
override could come from multiple places... if it comes from the highest priority then it is a force (set on the ORG, like turning off code saving)

But you could try to set an OVERRIDE environment variable to renable it,  That wont work since ENV is lower priority than ORG cloud seettings